src/main/java/gololang/Predefined.java

```diff
@@ -598,7 +603,7 @@ public static boolean isArray(Object object) {
    * @return the Character value.
    * @throws IllegalArgumentException if {@code obj} is not a number or a String.
    */
-  public static Object charValue(Object obj) throws IllegalArgumentException {
+  public static Object charValue(Object obj) {
```

Comment:
Just curious, why do you say it's wrong? Because of the documenting value of being explicit?
If so I agree with that point, although is in the javadoc :smile:
Anyway, I'll put it back. I've removed such explicit `throws` of `RuntimeExceptions` in several places, should I rollback everywhere?